src/components/KTreeList/KTreeItem.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div
```

Comment:
Should we make this an anchor tag so you can tab through the items?

Reply:
`<a>` with `role="button"` -- it will also need `href="#" and `@click.prevent="handleClick"`